cli/commands.go

```diff
@@ -185,7 +192,12 @@ func commonInit(
 }
 
 func doHandleBootstrapArtifact(config *conf.MenderConfig, opts *runOptionsType) error {
-	controller, mp, err := commonInit(config, opts)
+	_, err := os.Stat(config.BootstrapArtifactFile)
+	initDbOnly := false
+	if err != nil {
+		initDbOnly = true
+	}
```

Comment:
Making it dependent on the bootstrap file is not needed, IMHO, you can just pass `true` unconditionally in this function. The operations that need a full init do it again later anyway.

Reply:
sure. fixed.

Reply:
and also I tested manually to verify and be sure all is fine. the following scenarios:
* no db
* empty db
* db with only artifact_name unknown
* full db from real device
```
real	0m0.424s
user	0m0.380s
sys	0m0.044s
```
in all the cases.  (and yes, I have some ideas for the tests, just let me get some spare time after the cherry-picks)